pages.it/common/git-reset.md

```diff
@@ -14,7 +14,7 @@
 
 - Rimuovi dall'area di stage solo alcune porzioni di un file:
```

Comment:
```suggestion
- Rimuovi dall'area di stage solo alcune porzioni di un file in modo interattivo:
```
From the other italian pages it looks like the most common translation of `interactively` is just appending `in modo interattivo` to the sentence.

Reply:
Go for it, @glenacota 👍🏻'd it.